src/containers/InteractionButton/index.js

```diff
@@ -92,14 +91,14 @@ class InteractionButton extends Component {
 
     const btn = (
       <button className={classNames} type={type || 'button'} onClick={onClickHandler} disabled={isDisabled}>
-        <div className="interactionButton__inner">{children}</div>
+        <div className={cx('interactionButton__inner')}>{children}</div>
       </button>
     )
 
     if (isLoading) {
       return (
         <button className={classNames} type="button" disabled>
-          <div className="interactionButton__inner">{children}</div>
+          <div className={cx('interactionButton__inner')}>{children}</div>
```

Comment:
You use `__inner` here and above, but the related css uses `interactionButtonInner`